declarations/文心一言（Wenxinyiyan）.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "文心一言（Wenxinyiyan）",
```

Beware, there are hidden characters and spaces in the service name.

As documented in the [Contributing terms](https://docs.opentermsarchive.org/contributing-terms/) documentation page, along with [practical guidelines](https://docs.opentermsarchive.org/guidelines/declaring/) for specific cases.
At this stage, we recommend to use the service name as it is (Chinese characters) and use the transliteration in the service ID.

```suggestion
  "name": "文心一言",
```